src/Microsoft.Azure.AppConfiguration.AspNetCore/AzureAppConfigurationRefreshMiddleware.cs

```diff
@@ -24,9 +29,21 @@ public AzureAppConfigurationRefreshMiddleware(RequestDelegate next, IConfigurati
 
         public async Task InvokeAsync(HttpContext context)
         {
-            foreach (var refresher in Refreshers)
+            long refreshReadyTime = Interlocked.Read(ref _refreshReadyTime);
+
+            if (refreshReadyTime <= DateTimeOffset.UtcNow.Ticks && 
```

Comment:
DateTimeOffset.UtcNow.Ticks needs a variable, because it should be used in the new value of `_refreshReadyTime`

Reply:
Missed that, thanks